src/trixi_include.jl

```diff
@@ -45,7 +45,12 @@ function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
     if !mpi_isparallel(Val{:MPIExt}())
         @info "You just called `trixi_include`. Julia may now compile the code, please be patient."
     end
-    Base.include(ex -> replace_assignments(insert_maxiters(ex); kwargs...), mod, elixir)
+    Base.include(ex -> replace_assignments(insert_maxiters(mapexpr(ex)); kwargs...),
```

Comment:
Why did you choose this order of operations?

Reply:
I thought about it and couldn't come up with a reason for any order, so I just chose this one.

Reply:
👍